worldwind/src/commonMain/kotlin/earth/worldwind/draw/DrawableShape.kt

```diff
@@ -32,9 +31,38 @@ open class DrawableShape protected constructor(): Drawable {
         // TODO shape batching
         val program = drawState.program ?: return // program unspecified
         if (!program.useProgram(dc)) return // program failed to build
-        if (drawState.vertexBuffer?.bindBuffer(dc) != true) return  // vertex buffer unspecified or failed to bind
         if (drawState.elementBuffer?.bindBuffer(dc) != true) return  // element buffer unspecified or failed to bind
 
+        // Use the shape's vertex point attribute and vertex texture coordinate attribute.
+        dc.gl.enableVertexAttribArray(1 /*vertexTexCoord*/)
```

Comment:
Should be enabled only for elements in for (vertexAttribute in vertexBuffer.attributes)

Reply:
done